### ISSUE TITLE
Transhuman Forces Suggestion

### ISSUE THREAD

**Issue by glob245** [Minerva-Servers/minervahl2rp-suggestions#13]
Now.

Something i always wondered in HL2 and never really got an awnser to is how the Combine Soldiers can see your exact "coordinates"

EX. "Is at grid nine dash two"
Two parts of the Combine Soldier hud (Possibly Metrocop hud) that tell you YOUR current "coordinates" and the targets.

Example : 

TL - Grid Five Dash Six

UL - Grid Eight Dash Nine.

UL means Unit Location
TL means Target Location.

Basically just tells you TL via where your cursor is aiming, and UL by your player position. 

That's all for this suggestion.
